src/care/host_ptr.h

```diff
@@ -86,7 +86,7 @@ namespace care {
          ///
          template <bool B = std::is_const<T>::value,
                    typename std::enable_if<B, int>::type = 1>
-         host_ptr<T>(chai::ManagedArray<T_non_const> const &ptr) : m_ptr(ptr.getPointer(chai::CPU)) {}
+         host_ptr<T>(chai::ManagedArray<T_non_const> ptr) : m_ptr(ptr.getPointer(chai::CPU)) {}
```

Comment:
Likewise, add a TODO comment saying that this should be reverted to const in a future update of chai . That way, we can just grep TODO when we next update chai and not miss anything

Reply:
Done.